### PR TITLE
Enable builds without ceph-test subpackage

### PR DESCRIPTION
**Pull request by rjfd**
Fixes: http://tracker.ceph.com/issues/16776

Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
:+1: This is very useful for speeding up the build (and making it less resource-hungry) if you don't need the ceph-tests package.

**Comment by rjfd** [comment]
Rebased to resolve conflicts with yesterday commits.

File: ceph.spec.in [hunk 1]
```diff
@@ -1391,6 +1401,8 @@ ln -sf %{_libdir}/librbd.so.1 /usr/lib64/qemu/librbd.so.1
 %{python3_sitelib}/__pycache__/ceph_daemon.cpython*.py*
 %endif
 
+#################################################################################
```
Comment (smithfarm) [inline ceph.spec.in]: @rjfd Please drop this divider line (these divider lines were dropped by #9757)

**Comment by rjfd** [comment]
@smithfarm done with dropping

**Comment by smithfarm** [comment]
pushed `wip-16776` to gitbuilders

**Comment by smithfarm** [comment]
gitbuilders passed!

**Comment by smithfarm** [comment]
**running** http://pulpito.front.sepia.ceph.com:80/smithfarm-2016-08-28_16:42:29-upgrade:jewel-x-wip-16776---basic-smithi/

**Comment by smithfarm** [comment]
@b-ranto @ktdreyer @tchaikov Care to review?

**Comment by smithfarm** [comment]
Jenkins test this please

**Comment by tchaikov** [comment]
lgtm

**Comment by ktdreyer** [comment]
Would you please also move `ceph-client-debug` in `debian/ceph-base.install` to `debian/ceph-test.install` ?

**Comment by rjfd** [comment]
@ktdreyer done

**Comment by ktdreyer** [comment]
LGTM

**Comment by smithfarm** [comment]
@tchaikov What additional testing does this PR need?

**Comment by tchaikov** [comment]
@smithfarm better off run it through a rados run, just to make sure the nothing is missing in the ceph-test package.

**Comment by smithfarm** [comment]
ok, repushed `wip-16776` to pick up the latest commit

**Comment by smithfarm** [comment]
@rjfd Please rebase - there is a conflict.

**Comment by rjfd** [comment]
@smithfarm rebase done

**Comment by tchaikov** [comment]
@rjfd and @smithfarm will pull this PR into my test branch for the qa run.

**Comment by smithfarm** [comment]
@tchaikov Thanks!

File: src/tools/CMakeLists.txt [hunk 2]
```diff
@@ -46,9 +48,15 @@ if(WITH_LIBCEPHFS)
   install(TARGETS ceph-client-debug DESTINATION bin)
 endif(WITH_LIBCEPHFS)
 
+if(WITH_TESTS)
```
Comment (tchaikov) [inline src/tools/CMakeLists.txt]: @rjfd you might want to move this block into https://github.com/ceph/ceph/pull/10872/files#diff-65c9cee53e3f940aed5b8ad9942ff108L44 .

Comment (tchaikov) [inline src/tools/CMakeLists.txt]: see http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-deb-trusty-amd64-basic/log.cgi?log=33dbbbf9b7cf43326ebf29e322422df698116ff6

**Comment by rjfd** [comment]
@tchaikov thanks for catching that! I fixed it and pushed the new changes.

**Comment by smithfarm** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;SUwiDIsH&#x2F;1807-issue-16776-enable-rpm-build-without-ceph-test-subpackage"> (issue#16776) Enable RPM build without ceph-test subpackage</a>

**Comment by tchaikov** [comment]
tested at http://pulpito.ceph.com/kchai-2016-09-10_15:46:37-rados-wip-kefu-testing---basic-mira/408471/

the failed test of http://pulpito.ceph.com/kchai-2016-09-10_15:46:37-rados-wip-kefu-testing---basic-mira/408471/ is irrelevant.

**Pull request merged**